admin_guide/diagnostics_tool.adoc

```diff
@@ -45,10 +45,75 @@ administrator, run:
 $ oc adm diagnostics
 ----
 
-This runs all available diagnostics, skipping any that do not apply. For
-example, the *NodeConfigCheck* does not run unless a node configuration is
-available. You can also run specific diagnostics by name as you work to address
-issues. For example:
+This runs all available diagnostics, skipping any that do not apply. 
+
+You can also run the diagnostics tool on an individual element:
```

Comment:
> I like having the table of specific diagnostics below. I would suggest removing the list from the Overview as the table lists out the same things by which diagnostic does them.

I was thinking the same :) I wanted to wait before we had the table going before I did in case I got rid of anything that was needed.